codespell_lib/data/dictionary.txt

```diff
@@ -5308,6 +5310,7 @@ doesent'->doesn't
 doesent->doesn't
 doesing->doing, does, does in, dosing, dozing,
 doesn;t->doesn't
+doesnexist->doesntexist
```

Comment:
This should correct to `doesn't exist`, but I can't remember if we support multiple words in the correction currently.

Reply:
done